### PR TITLE
create contact_us

### PR DESCRIPTION
**Pull request by boikonataliii**
**STATUS**
READY

**DESCRIPTION**
Contact us form.

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [x] code coverage >90%
- [x] DB indexes added if needed
- [x] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs

### CODE REVIEW

**Comment by boikonataliii** [comment]
screenshot from 2018-11-26 04-43-13
screenshot from 2018-11-26 04-45-31
screenshot from 2018-11-26 04-45-52

File: db/migrate/20181125214849_create_contacts.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateContacts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :contacts do |t|
+      t.string :name
+      t.string :email
+      t.string :phone
+      t.string :message
```
Comment (BohdanBusko) [inline db/migrate/20181125214849_create_contacts.rb]: зміни тип на text

File: spec/controllers/contacts_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe ContactsController, type: :controller do
+
+  let!(:valid_contact) { FactoryBot.create(:contact) }
+
+  describe "GET #new" do
+    it "returns a 200 status code" do
+      get :new
+      expect(response).to have_http_status(200)
```
Comment (BohdanBusko) [inline spec/controllers/contacts_controller_spec.rb:10]: be_successful теж працює

File: app/views/contacts/new.html.slim [hunk 3]
```diff
@@ -0,0 +1,8 @@
+- title "Contact us"
+.hero
+  .hero-body
+    .container.has-text-centered
+      .column.is-6.is-offset-3.box
+        .title Contact us
+        #form_contact
+          = render(partial: 'contacts_form', object: @contact)
```
Comment (igorkasyanchuk) [inline app/views/contacts/new.html.slim]: specify full path to partial
and rename it to "form"

**Review by igorkasyanchuk — APPROVED** [review]
approved with minor comment

**Pull request merged**